website/src/pages/documents/document.page.tsx

```diff
@@ -239,19 +240,36 @@ const DocumentContents = ({
   levelOfDetail: LevelOfDetail
   cherokeeRepresentation: Dailp.CherokeeOrthography
   openDetails: (morpheme: any) => void
-  wordPanelDetails: WordPanelDetails
+  wordPanelDetails: PanelDetails
 }) => {
-  const [{ data }] = Dailp.useDocumentContentsQuery({
+  const [result, rerunQuery] = Dailp.useDocumentContentsQuery({
     variables: {
       slug: doc.slug,
       isReference: doc.isReference,
       morphemeSystem: cherokeeRepresentation,
     },
   })
-  const docContents = data?.document
+
+  const docContents = result.data?.document
+  const { form, isEditing } = useForm()
+
+  useEffect(() => {
+    // If the form has been submitted, update the panel's current contents to be the currently selected word
```

Comment:
I'm not sure if this is working. When I make edits I find that I have to reload the page in order for the changes to appear.

Reply:
That's odd.. It's working on my local, but which ones are you editing that don't update? Also, @loafofpiecrust are you having this issue as well?

Reply:
Ah, nvm, that part works.